infra/locals.tf

```
@@ -0,0 +1,3 @@
+locals {
+  project = "${var.prefix}${random_id.unique.hex}-${var.env_short}-${var.location_short}"
```

```suggestion
  project = "${var.prefix}-${var.env_short}-${var.location_short}"
```
random id not required